en/translation.json

```diff
@@ -332,6 +332,9 @@
   "this_transaction_not_events": "This transaction does not have events",
   "unable_locate_message_transaction": "Unable to locate messages for transaction",
   "this_transaction_not_messages": "This transaction does not have messages",
+  "this_transaction_no_selected_messages" : "This transaction has no messages of selected type",
```

Comment:
Keys should contain at most four words (according to [guidelines](https://www.notion.so/nethermind/Voyager-Translations-4d1088b3e14f4a9b94ac5640a11c9396?pvs=4#bca57930dcd24ed189a63c8e08df70a4)).
Some keys may violate this rule already; we need to confirm if we're still adhering to the four-word key rule.
@prix0007 can you confirm?